functions/taskrouterListeners/postSurveyListener.private.ts

```diff
@@ -78,7 +78,8 @@ const isTriggerPostSurvey = (
  * Checks the event type to determine if the listener should handle the event or not.
  * If it returns true, the taskrouter will invoke this listener.
  */
-export const shouldHandle = (event: EventFields) => eventTypes.includes(event.EventType);
+export const shouldHandle = () => false;
```

Comment:
Will this not mean this code will never run, whether the flag is enabled or not?

Reply:
Ups, thanks for the catch, just reverted that.